### ISSUE TITLE
Ignore/Exclude on Windows

### ISSUE THREAD

**Issue by marcus-at-localhorst** [wakatime/sublime-wakatime#70]
Hi,
I struggle to set the ignore paths in .wakatime.cfg and/or WakaTime.sublime-settings on Windows.
Whatever pattern I use, the files get tracked.

```json
{
	"api_key": "xxx",
	"ignore": [
		"e:/Test/",
		".*/Dropbox/",
		"e:/LABEL/*",
		"e:/WEB/@docs/*",
		"E:\\WEB\\@docs\\*",
		".*/xampp/*"
	],
	"debug": false
}
```

I would expect, that the pattern is platform independent and catches partial strings, like
`/WEB/` catches `e:/WEB/sub/test.php` or `f:/sub/WEB/index.html`.

So what is the correct exclude pattern on Windows for partial paths?
Can I see if it works in the sublime console/.wakatime.log?
Which priority have settings in .wakatime.cfg and WakaTime.sublime-settings ?

Thanks, Marcus

**Comment by alanhamlett** [comment]
> Can I see if it works in the sublime console/.wakatime.log?
Set debug to `true` and it will write to the log file whenever a file is ignored.

> So what is the correct exclude pattern on Windows for partial paths?
I would use this pattern without the drive letter to make sure you match forward and backward slashes:
```
	"ignore": [
		"[\\\\/]Test[\\\\/]",
		"[\\\\/]Dropbox[\\\\/]",
		"[\\\\/]LABEL[\\\\/]",
		"[\\\\/]WEB[\\\\/]@docs[\\\\/]",
		"[\\\\/]WEB[\\\\/]@docs[\\\\/]",
		"[\\\\/]xampp[\\\\/]"
	],
```

The regex follows POSIX standard so `*` means zero or more of the previous character, which is different from `*` as a wildcard in bash terms. To match any character like `*` in bash, use `.*`. However, the regex only needs to match part of the path not the full path so `.*` at the start or end of an ignore pattern doesn't do anything.

**Comment by marcus-at-localhorst** [comment]
I traced it down.

in `~/.wakatime.cfg` using the recommended [configuration](https://github.com/wakatime/wakatime#configuring)

```ini
exclude =
  \\xampp\\
  [\\/]xampp[\\/]
```
both versions work fine

But for the sublime-settings the following pattern is just *ignored*

```js
{
  "ignore": [
    "\\\\xampp\\\\",
    "[\\\\/]xampp[\\\\/]"
  ]
}
```
Digging through the source code and looking up the command line parameters made it clear that the `--exclude` parameter is never called :)

https://github.com/wakatime/sublime-wakatime/blob/master/WakaTime.py#L539

It looks like `ignore` was used before, but wasn't changed [later](https://github.com/wakatime/wakatime/commit/ef6342f16fd1656a2acf4eebc42381aa8b5df003) in the the sublime text version.

Would it be possible to make all `~/.wakatime.cfg` options available in the `WakaTime.sublime-settings` file ?

**Comment by alanhamlett** [comment]
Fixed with #71.
